x-pack/filebeat/module/iptables/log/test/geo.log-expected.json

```diff
@@ -3,20 +3,32 @@
         "destination.ip": "10.4.0.5",
         "destination.mac": "90:10:20:76:8d:20",
         "destination.port": 443,
+        "event.action": "d",
```

Comment:
Looks like this should be `deny` or something more than `d`.

Reply:
This is implementing @webmat 's suggestion.  The original value from ubiquiti iptables is "d" or "a"

Reply:
I think expanding it to a word will be more user friendly. I have a feeling users will think it's some kind of bug like me.

Reply:
sure.  switched to accept/drop which is what is displayed in ubiquiti gui

Reply:
Agree with switching to being more readable.

The ECS recommendation is only a high level for "this should be driven by the source". But when the underlying source transport is too compact like this "a" & "d", I think it's totally fine to expand it. Expanding it to what's in their UI is actually a great resolution here.